build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -14,7 +14,7 @@
             DependsOnTargets="SetupNuGetPackagesArchiveInputsOutputs;
                               SetupAzureBlobInformation;
                               GenerateNuGetPackagesArchive"
-            Condition=" '$(UploadNuGetPackagesArchiveToAzure)' == 'true' And  '$(PUBLISH_TO_AZURE_BLOB)' != '' ">
+            Condition=" '$(UploadNuGetPackagesArchiveToAzure)' == 'true' And  '$(PUBLISH_LZMA_TO_AZURE_BLOB)' != '' ">
```

Comment:
I haven't been following this change but wanted to make sure this property name change was intentional